spring-core-2/src/main/java/com/baeldung/pojo/EmployeePojo.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.pojo;
+
+import java.time.LocalDate;
+
+public class EmployeePojo {
+
+    public String firstName;
+
+    public String lastName;
+
+    private LocalDate startDate;
+
+    public EmployeePojo(String firstName, String lastName, LocalDate startDate) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.startDate = startDate;
+    }
+
+    String name() {
+        return this.firstName + " " + this.lastName;
+    }
+
+    LocalDate getStart() {
+        return this.startDate;
+    }
```

Comment:
These methods should be `public` rather than `package-private`

Reply:
Or is the point that they're not `public` so it's not a bean?

Reply:
@ashleyfrieze I have corrected these to be public now.